jestProcess.js

```diff
@@ -0,0 +1,10 @@
+const babelOptions = {
+  presets: ["env", "stage-0", "react"],
```

Comment:
Stage-0 seems like it's a bit much to me, do you know what JS feature exactly this is adding that the tests need?

I did try running without stage-0 and it didn't work.